package.json

```diff
@@ -7,20 +7,22 @@
     "mysql": "^2.11.1"
   },
   "devDependencies": {
-    "babel": "^6.1.18",
+    "babel": "^6.23.0",
+    "babel-cli": "^6.24.1",
```

Comment:
I would agree that we should not use global dependencies for this. Since we don't have a plan to clearly define our build/dev environment for all projects yet, making this depend on global would prob create other issues.